config/create_fortran_test.py

```diff
@@ -0,0 +1,138 @@
+#
```

Comment:
This entire comment should also be in the project readme because that's where a potential contributor would look first.

Reply:
Added text in TESTS.md